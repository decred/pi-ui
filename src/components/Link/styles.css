.link {
  color: var(--color-primary);
}
