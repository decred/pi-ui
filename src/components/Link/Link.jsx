import React from "react";
import PropTypes from "prop-types";
import styles from "./styles.css";

const Link = ({ href, children }) => {
  return (
    <a className={styles.link} href={href}>
      {children}
    </a>
  );
};

Link.propTypes = {
  href: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired
};

export default Link;
