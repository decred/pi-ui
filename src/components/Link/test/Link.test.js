import React from "react";
import Link from "../Link";
import { create } from "react-test-renderer";

describe("Link component", () => {
  test("Matches the snapshot", () => {
    const link = create(<Link>test</Link>);
    expect(link.toJSON()).toMatchSnapshot();
  });
});
