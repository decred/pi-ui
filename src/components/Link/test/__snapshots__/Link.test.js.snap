// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component Matches the snapshot 1`] = `
<a
  className="link"
>
  test
</a>
`;
